packages/playground/src/layouts/RootLayout.tsx

```diff
@@ -85,9 +85,9 @@ const RootLayout: React.FC = () => {
   ]
 
   return (
-    <div className="bg-background flex">
+    <div className="bg-background grid md:grid-cols-[220px_minmax(900px,_1fr)]">
```

Comment:
maybe we should also add the class `min-w-screen` to ensure a min-width of 100vh, incase the content doesn't have enough width